scripts/keystone_data.sh

```diff
@@ -194,9 +194,9 @@ if [[ "$ENABLED_SERVICES" =~ "n-cpu" ]]; then
     get_or_create_endpoint \
         "placement" \
         "RegionOne" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/"
+        "http://$SERVICE_HOST:8778/" \
```

Comment:
It looks like the install guide stopped using ports in the endpoints so it doesn't really matter any more what port this is running on https://review.openstack.org/#/c/450009/